tidb/upgrade.bash

```diff
@@ -0,0 +1,35 @@
+set -euo pipefail
+. "`cd $(dirname ${BASH_SOURCE[0]}) && pwd`/../helper/helper.bash"
+
+env=`cat "${1}/env"`
+shift
+
+confirm=`confirm_str "${env}"`
+name=`must_env_val "${env}" 'tidb.cluster'`
+ver=`must_env_val "${env}" 'tidb.version'`
+
+shift 3
+
+ver_path=`expr "${ver}" : '\(.*+\)' || true`
```

Comment:
Have the same logic code block in other place.
Make it a helper function in helper.bash?